terraform/031-email-service/main.tf

```
@@ -90,6 +135,7 @@ module "ecsservice_api" {
   container_def_json = "${data.template_file.task_def_api.rendered}"
   desired_count      = "${var.desired_count_api}"
   tg_arn             = "${aws_alb_target_group.email.arn}"
+  task_role_arn      = "${aws_iam_role_policy.ses.id}"
```

I think this should be the role arn, not the policy right?
```suggestion
  task_role_arn      = "${aws_iam_role.ses.id}"
```